rust/query-stats/README.md

```diff
@@ -0,0 +1,75 @@
+# Query stats
+
+## Prerequisites
+This example requires an installation of:
+
+- [x] Install the [IC SDK](../developer-docs/setup/install/index.mdx) at version 0.16.1 or higher.
+- [x] Download the following project files from GitHub: https://github.com/dfinity/examples/
+
+Begin by opening a terminal window.
+
+This is currently only for use from within DFINITY, as the replica shipped with `dfx` as well as nodes on 
+mainnet do not enable the query stats feature at the moment.
```

Comment:
If this is true this should not be in this repo

Reply:
This is currently true, but the canary rollout should start soon.